Reports/reportModels.js

```diff
@@ -0,0 +1,26 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var eventReportSchema = new Schema({
+    type: String,
+    approvalStatus: String,
+    student: String,
+    event: String,
+})
+
+var otherReportSchema = new Schema({
+    type: String,
+    approvalStatus: String,
```

Comment:
Boolean is part of [BSON](http://bsonspec.org/spec.html) which is what MongoDB uses. Since we are using these schemas to force types we might want to use that.